cmake/gtwrapConfig.cmake

```diff
@@ -10,15 +10,9 @@ set(GTWRAP_DIR "${CMAKE_CURRENT_LIST_DIR}")
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
 
-if(NOT CMAKE_PREFIX_PATH)
-  set(GTWRAP_BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
-  set(GTWRAP_LIBDIR ${CMAKE_INSTALL_FULL_LIBDIR})
-  set(GTWRAP_INCLUDEDIR ${CMAKE_INSTALL_FULL_INCLUDEDIR})
-else()
-  set(GTWRAP_BINDIR "${CMAKE_PREFIX_PATH}/bin")
-  set(GTWRAP_LIBDIR "${CMAKE_PREFIX_PATH}/lib")
-  set(GTWRAP_INCLUDEDIR "${CMAKE_PREFIX_PATH}/include")
-endif()
+set(GTWRAP_BINDIR ${GTWRAP_DIR}/../../../bin)
```

Comment:
Is this always true? What about OSes like Debian where it segregates the lib to lib or lib64 depending on the architecture type?

Reply:
I think the answer is no. What is the root cause of this PR?